src/ripple/app/consensus/RCLConsensus.cpp

```diff
@@ -210,12 +210,21 @@ RCLConsensus::Adaptor::propose(RCLCxPeerPos::Proposal const& proposal)
         proposal.prevLedger().begin(), proposal.prevLedger().size());
     prop.set_proposeseq(proposal.proposeSeq());
     prop.set_closetime(proposal.closeTime().time_since_epoch().count());
+
+    if(!validatorKeys_.keys)
+    {
+        JLOG(j_.fatal()) << "RCLConsensus::Adaptor::propose: ValidatorKeys "
```

Comment:
I think that logging this as `fatal()` is overkill.  I suggest `error()` or even `warn()`.  If a bug gets in that triggers this message then the logs will be swamped with this log message.  At `fatal()` level it is hard to remove the message.  And the recovery, simply returning, is benign.

Reply:
okay